string target_name
---
MarkerDimensions dimensions
